tools/kl_divergence/lm_head_utils.py

```
@@ -0,0 +1,206 @@
+
+        try:
+            index_path = hf_hub_download(
+                model_path, "model.safetensors.index.json"
+            )
+            with open(index_path) as f:
+                index = json.load(f)
+            return {
+                name: (model_path, shard)
+                for name, shard in index["weight_map"].items()
+            }
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)